data/strings/strings.txt

```diff
@@ -1748,7 +1750,7 @@
     ro = Descarci %@?
     ru = Загрузить %@?
     sk = Stiahnuť %@?
-    sr = Преузети %@?
+    sr = Преузимате %@?
```

Comment:
Is it a noun or a verb? Should be a verb.

Reply:
It's a verb. Because a question was asked.
Преузимате / "%@" I assume it's a map
**Преузимате Serbia/Србија?**

Anyway. This question exists in all navigations and is grammatically incorrect as far as the Serbian language is concerned.

Map name is 'fixed' СРБИЈА. And there is no change. The correct sentence would be:

Преузимате Србиј**У**?
or
Преузимате мапу Србиј**Е**? 

He can't write at all "СРБИЈА"